src/ffi.rs

```diff
@@ -848,12 +901,12 @@ pub extern fn quiche_stream_iter_free(iter: *mut StreamIter) {
 
 #[repr(C)]
 pub struct Stats {
-    pub recv: usize,
-    pub sent: usize,
-    pub lost: usize,
-    pub rtt: u64,
-    pub cwnd: usize,
-    pub delivery_rate: u64,
+    recv: usize,
+    sent: usize,
+    lost: usize,
+    rtt: u64,
+    cwnd: usize,
+    delivery_rate: u64,
```

Comment:
why is this required in this commit?

Reply:
It's not, really, I just noticed this struct wasn't consistent with the others, and I thought it doesn't really warrant its own PR.